src/Config/CliBuilder.ts

```diff
@@ -89,12 +94,33 @@ export class CliBuilder {
       .reduce((acc: Argv, item: CommandModule) => {
         const handler = item.handler.bind(item);
 
-        item.handler = (args) =>
-          runWithAsyncContext(() => {
-            setContext('args', args);
+        item.handler = async (args: Arguments) => {
+          const systemConfigReader = new SystemConfigReader(args.api as string);
+          const systemConfig = await systemConfigReader.read();
 
-            handler(args);
+          initSentry({
+            attachStacktrace: true,
+            dsn: systemConfig.sentryDsn,
+            release: process.env.VERSION,
+            beforeSend: (event) => {
```

Comment:
Utilize shorthand methods when there's no need to reference instance members:
```suggestion
            beforeSend(event) {
```